libunwind/src/DwarfInstructions.hpp

```diff
@@ -165,9 +165,10 @@ typename A::capability_t DwarfInstructions<A, R>::getSavedCapabilityRegister(
 #else
     break;
 #endif
+  case CFI_Parser<A>::kRegisterUndefined:
+    return fake_capability_t();
```

Comment:
Actually, can we just use `A::to_capability_t(0)` here? A lot of this code could do with some cleanup but since that function already exists, we might as well use it here.

Reply:
Do we want to use `A::to_capability_t(0)` or `addressSpace.to_capability_t(0)` as a style? libunwind seems to predominantly use the latter for other things.

Reply:
I guess we should match the common style, so I guess the latter is better.